packages/web-runtime/src/pages/accessDenied.vue

```diff
@@ -82,7 +82,6 @@ export default defineComponent({
         }
       }
       return {
-        name: 'login',
```

Comment:
No idea why we had the `name` key here... maybe just by mistake and it was really meant for the `to` route?!